src/classes/MTCH_FindGifts_CTRL.cls

```diff
@@ -27,13 +27,13 @@
     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
     POSSIBILITY OF SUCH DAMAGE.
 */
-/*
+/**
```

Comment:
You should either make this one match the other comments with a long string of *, or just leave it as is and remove this change.